docs: app access and auto-discovery guides: friction points found during agent testing (#69352)
gravitational/teleport: docs/pages/enroll-resources/application-access/cloud-apis/awsoidc-integration.mdx

### Validate the integration

> To validate the integration without making a mutating AWS change, run:

Reviewer: ```suggestion
To validate the integration without modifying any AWS resources, run:
```
Nit, suggested for specificity and reducing redundancy. Feel free to leave this alone or use a more accurate wording depending on what "mutating AWS change" means.

Author: I like your version better!

```code
$ tctl integrations test <Var name="my-integration"/>
```

> This command calls AWS STS `GetCallerIdentity` using the OIDC token exchange, confirming that AWS trusts Teleport's identity provider and can assume the configured IAM role. If the integration is working, the command returns the AWS account ID and assumed role ARN, and you can skip ahead to [Next steps](#next-steps).

Reviewer 2: Nit: wrap the long line.

Author: wrapped

### Validate the integration

> To validate the integration without modifying any AWS resources, run::

Reviewer 3: ```suggestion
To validate the integration without modifying any AWS resources, run:
```
